core/templates/dev/head/pages/base.html

```diff
@@ -119,6 +119,10 @@
       {% endif %}
     </script>
 
+    <!-- Note to developers: This script needs to be loaded before external
```

Comment:
I would still need the 'load_constants.js' file for karma tests, should I move it to 'tests' folder and call it something like 'karma-constants.js'?